spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-functional-binder-sample/spring-cloud-gcp-pubsub-stream-functional-binder-sample-sink/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+# Binding name is mapped to a simpler shortcut name "input".
+spring.cloud.stream.function.bindings.convertUserMessageToString-in-0=input
```

Comment:
Then you have to refer to `convertUserMessageToString-in-0` everywhere else. We don't have to do it, but it's a new feature in SCS, so it might be nice to leave as an option. I added it into the sink and not into the source because the source has zero other configuration, while the sink also has the group subscription name.

So, without aliasing, configuration looks like this:
```
spring.cloud.stream.bindings.logUserMessage-in-0.destination=my-topic
spring.cloud.stream.bindings.logUserMessage-in-0.group=my-group
```

And with aliasing, it looks like this:
```
spring.cloud.stream.function.bindings.logUserMessage-in-0=input
spring.cloud.stream.bindings.input.destination=my-topic
spring.cloud.stream.bindings.input.group=my-group
```

This looks more dramatic once your chain -- function name becomes something like `logUserMessage-in-0|processUserMessage-in-0`.